dapp/src/utils/bridge.js

```diff
@@ -13,23 +15,45 @@ export const getBridgeMediator = (bridgeType, networkType) => {
   return bridgeMediator
 }
 
-export function approveToken({ networkType, tokenAddress, amount, bridgeType = 'home' }, {
+export const homeToForeignBridgeMediators = {
+  fuse: isProd
```

Comment:
Can we move that logic to the CONFIG? 
having fields like:
`CONFIG.web3.bridge.addresses.homeToForeign.home` and CONFIG.web3.bridge.addresses.homeToForeign.foreign`